rs/nns/governance/src/lib.rs

```diff
@@ -204,6 +206,20 @@ thread_local! {
     static ACTIVE_NEURONS_IN_STABLE_MEMORY_ENABLED: Cell<bool> = const { Cell::new(false) };
 }
 
+pub fn is_prune_following_enabled() -> bool {
+    IS_PRUNE_FOLLOWING_ENABLED.with(|ok| ok.get())
+}
+
+/// Only integration tests should use this.
+pub fn temporarily_enable_prune_following() -> Temporary {
```

Comment:
I believe it's possible to put these "temporarily_" methods behind feature flags.  I did so in a different PR: https://github.com/dfinity/ic/pull/2474/files

Reply:
Done.